mxcube3/ui/components/SampleQueue/HistorySampleItem.js

```diff
@@ -6,17 +6,17 @@ import './app.less';
 export default class HistorySampleItem extends React.Component {
   constructor(props) {
     super(props);
-    this.collapseSample = props.collapseSample.bind(this, props.id);
+    this.collapseSample = props.collapseSample.bind(this, props.data.clientID);
```

Comment:
Wow wow, the data.clientID should not be there Ill better take a look at that.

Thanks